### PR TITLE
CORE-1450 Update to model v5.2

### PR DESCRIPTION
**Pull request by psarando**
This PR will update the `model` package to `v5.2` with the fix from cyverse-de/model#20.

This required updating the import paths `gopkg.in/cyverse-de/model` and `gopkg.in/cyverse-de/messaging` to `github.com/cyverse-de/model` and `github.com/cyverse-de/messaging` for `go mod`.

### CODE REVIEW

**Review by slr71 — CHANGES_REQUESTED** [review]
I don't think this is going to work the way it is, unfortunately. Switching to Go modules changes the import path from `gopkg.in/cyverse-de/model.v5` to `github.com/cyverse-de/model`, which will require us to change the import path. Unfortunately, that's going to require us to change other repositories as well because the import path has to be updated in other dependencies of `road-runner` that also use `model`. So far, I think that `model` and `messaging` are the only two that we'll have to update.

**Comment by slr71** [comment]
Oh, one thing I meant to mention in my review comment was that I haven't been able to find a way to get `go get` to fully update a dependency from `gopkg.in`. If we could get that to work then we could avoid sweeping changes.

File: status.go [hunk 1]
```diff
@@ -3,10 +3,17 @@ package main
...
+	"github.com/cyverse-de/messaging"
+	"github.com/cyverse-de/model"
 )
 
+func jobDetailsFromJob(job *model.Job) messaging.JobDetails {
+	return messaging.JobDetails{
+		InvocationID: job.InvocationID,
+		CondorID:     job.CondorID,
+	}
+}
```
Comment (psarando) [inline status.go:15]: Thanks to @slr71 for this suggestion 👍

**Review by slr71 — APPROVED** [review]
Looks good to me! 👍 :shipit:

**Comment by psarando** [comment]
Thanks for the review! 👍

**Pull request merged**